client/app/components/saveable-form/field/radio-group.hbs

```diff
@@ -1,21 +1,23 @@
-<@parentField
-  @type="radio"
-  @hideError={{true}}
-  as |RadioButton|
->
-  {{#each @options as |radio|}}
-    <span class="nowrap">
-      <RadioButton
-        @targetValue={{radio.code}}
-        @changed={{@onChange}}
-        data-test-radio={{@attribute}}
-        data-test-radio-option={{radio.label}}
-        ...attributes
-      >
-        {{radio.label}}
-      </RadioButton>
-    </span>
-  {{/each}}
-</@parentField>
+{{#let @parentField as |Field|}}
+  <Field
+    @type="radio"
+    @hideError={{true}}
+    as |RadioButton|
```

Comment:
This is a bit confusing to me. Can we just directly use the Radio component here?

Reply:
What's the pros/cons of accessing Radio through Field vs directly, like
`<Package::SaveableForm::Field:Radio />`?
One observation: we wouldn't have to specify `@hideError={{true}}` in that case.

Reply:
All this change does is use `let` to alias the component so that it's cleaner syntax? You're commenting about code that's already merged. I'm confused why this blocks an approval. But, here we go:

> This is a bit confusing to me. Can we just directly use the Radio component here?
It already does that. Could you clarify?

> What's the pros/cons of accessing Radio through Field vs directly, like
> `<Package::SaveableForm::Field:Radio />`?
I mean, do you really think invoking components directly like that is a better design? Accessing it directly would require passing all arguments every single time. That'll be like 5 arguments. We would have the same number of lines of code before we refactored. Using contextual components prevents that and is more semantic. Are you proposing a different API? What alternatives would you like to see?

> One observation: we wouldn't have to specify `@hideError={{true}}` in that case.
In what case?

Reply:
Hmm just FYI, all this change does is use `let` to alias the component so that it's cleaner syntax.

> This is a bit confusing to me. Can we just directly use the Radio component here?

It already does that. Could you clarify?

> What's the pros/cons of accessing Radio through Field vs directly, like
> `<Package::SaveableForm::Field:Radio />`?

I mean, are you saying that invoking directly is a better design? Accessing it directly would require passing all arguments every single time. That'll be like 5 arguments. We would have the same number of lines of code before we refactored. Using contextual components prevents that and is more semantic. Are you proposing a different API? What alternatives would you like to see?

> One observation: we wouldn't have to specify `@hideError={{true}}` in that case.

In what case?